scripts/rv_expression_association/subset_variants.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# pylint: disable=no-value-for-parameter,missing-function-docstring,missing-module-docstring
+
+import click
+import hail as hl
+from cpg_utils.hail_batch import dataset_path, init_batch, output_path
+
+
+@click.command()
+@click.option('--input-mt-path', required=True)  # 'mt/v7.mt'
+@click.option('--genes', required=True, help='List of genes to consider. Space separated, as one argument.')  # 'LMNA'
+@click.option('--output=mt-name', required=True)  # 'significant_genes_burden_max.mt'
```

Comment:
Is this supposed to have the equals sign?

Reply:
nope that's embarrassing and should be fixed now lol